vars/publishToRepository.groovy

```diff
@@ -35,6 +35,12 @@ def call(Map config = [:]) {
         println('No REPOSITORY_URL environment variable configured.')
         return
     }
+
+    // Don't publish from PRs
+    if (env.CHANGE_ID) {
```

Comment:
There are some cases where we want to publish from a PR, usually into a test repository, such as the self test for this module.

Reply:
Hrm.  How did the tests pass then?

Reply:
Tests are actually only run / graded if the test parameter is passed to the publishToRepositorySystem if that routine exists and the repository is not an APT repository.

Reply:
But the tests were run for this PR.